src/routes/common/transactions.helper.ts

```diff
@@ -16,6 +27,106 @@ const isCalldata = (data: string, signature: string): boolean => {
+      encodedData,
+    );
+
+  return {
+    singleton,
+    initializer,
+    saltNonce,
+  };
+};
+
+/**
+ * proxyCreationCode is hardcoded so that we don't have to query the blockchain for it
+ * Whenever a new version is deployed or we support relaying on more networks this will
+ * need to be updated or we revert to querying the blockchain for it.
+ *
+ * If we decide to query the blockchain, we should also consider using the core SDK instead.
+ *
+ * @see https://github.com/safe-global/safe-contracts/blob/main/contracts/proxies/SafeProxyFactory.sol#L15
+ */
+const PROXY_CREATION_CODE_VERSION = '1.3.0';
+const PROXY_CREATION_CODE: { [key in SupportedChainId]: string } = {
+  [SupportedChainId.GOERLI]:
+    '0x608060405234801561001057600080fd5b506040516101e63803806101e68339818101604052602081101561003357600080fd5b8101908080519060200190929190505050600073ffffffffffffffffffffffffffffffffffffffff168173ffffffffffffffffffffffffffffffffffffffff1614156100ca576040517f08c379a00000000000000000000000000000000000000000000000000000000081526004018080602001828103825260228152602001806101c46022913960400191505060405180910390fd5b806000806101000a81548173ffffffffffffffffffffffffffffffffffffffff021916908373ffffffffffffffffffffffffffffffffffffffff1602179055505060ab806101196000396000f3fe608060405273ffffffffffffffffffffffffffffffffffffffff600054167fa619486e0000000000000000000000000000000000000000000000000000000060003514156050578060005260206000f35b3660008037600080366000845af43d6000803e60008114156070573d6000fd5b3d6000f3fea2646970667358221220d1429297349653a4918076d650332de1a1068c5f3e07c5c82360c277770b955264736f6c63430007060033496e76616c69642073696e676c65746f6e20616464726573732070726f7669646564',
+  [SupportedChainId.GNOSIS_CHAIN]:
+    '0x608060405234801561001057600080fd5b506040516101e63803806101e68339818101604052602081101561003357600080fd5b8101908080519060200190929190505050600073ffffffffffffffffffffffffffffffffffffffff168173ffffffffffffffffffffffffffffffffffffffff1614156100ca576040517f08c379a00000000000000000000000000000000000000000000000000000000081526004018080602001828103825260228152602001806101c46022913960400191505060405180910390fd5b806000806101000a81548173ffffffffffffffffffffffffffffffffffffffff021916908373ffffffffffffffffffffffffffffffffffffffff1602179055505060ab806101196000396000f3fe608060405273ffffffffffffffffffffffffffffffffffffffff600054167fa619486e0000000000000000000000000000000000000000000000000000000060003514156050578060005260206000f35b3660008037600080366000845af43d6000803e60008114156070573d6000fd5b3d6000f3fea2646970667358221220d1429297349653a4918076d650332de1a1068c5f3e07c5c82360c277770b955264736f6c63430007060033496e76616c69642073696e676c65746f6e20616464726573732070726f7669646564',
+};
+
+/**
+ * Predicts the Safe address from createProxyWithNonce calldata
+ * @param chainId chainId
+ * @param to proxyFactory address
+ * @param data createProxyWithNonce calldata
+ * @returns Safe address
+ */
+export const predictSafeAddress = (
```

Comment:
What's the reason for this not to be just a function?

Reply:
After discussing this we agreed to change arrow functions to be regular functions across the entire project before we "finish" it. (Use of arrow functions is a habit from frontend development).